src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -158,7 +158,6 @@ public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePa
                 foreach (Fix fix in result.Fixes)
                 {
                     FixModel fixModel = fix.ToFixModel(run.OriginalUriBaseIds, regionsCache);
-                    fixModel.FixApplied += () => IsFixed = true;
```

Comment:
>FixApplied [](start = 29, length = 10)

This code was intended to avoid the light bulb once the error was fixed, but it didn't work. The logic for this (which now does work) is now in `FixSuggestedActionsSource`.